packages/shared/src/components/ExtensionOnboarding.tsx

```diff
@@ -25,15 +25,15 @@ const ExtensionOnboarding = (): ReactElement => {
       <Link href={onboardingUrl} passHref>
         <Button
           tag="a"
-          className="w-full btn-primary max-w-[18.75rem]"
+          className="w-full btn-primary max-w-[18.75rem] z-1"
           buttonSize={ButtonSize.Large}
         >
           Continue ➔
         </Button>
       </Link>
 
       <img
-        className="absolute bottom-0 w-[33rem]"
+        className="absolute bottom-0 w-[33rem] z-0"
```

Comment:
```suggestion
        className="absolute bottom-0 z-0 w-[33rem]"
```

Or: you don't really need the z-0 here though, it will work without.